examples/sample/opo/embed.conf

```diff
@@ -237,7 +237,7 @@ handler.embed.class = SampleController
 # support calling the embedded interpreter.
 
 embed.load_path = .,lib,examples/sample
-embed.require = sample
+embed.require = sample_controller
```

Comment:
you could have pushed this directly to the `develop` branch..

Reply:
I thought about it but figured I'd give you a chance to review.